jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/gradle/GradleRawConfigurationTest.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2018 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.gradle;
+
+import com.google.cloud.tools.jib.plugins.common.AuthProperty;
+import com.google.common.collect.ImmutableMap;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import org.junit.Assert;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+/** Test for {@link GradleRawConfiguration}. */
+public class GradleRawConfigurationTest {
+
+  @Test
+  public void testGetters() {
```

Comment:
From another perspective, this is testing "when Gradle is configured with these raw configuration values (values from `JibExtension`), do `RawConfiguration` getters return correct values?" Actually, I added these tests with that intention. It's also meaningful to catch errors from copy-and-paste (e.g., some getter returning a value of a different field.) I think using a mock is our best option, since we don't want to introduce setters to `JibExtension` or make its field visible just for testing.